users/autostart.rst

```diff
@@ -137,40 +137,38 @@ browser to interact with Syncthing).
 
 .. seealso:: :ref:`Windows GUI Wrappers <contrib-windows>`, :ref:`Cross-platform GUI Wrappers <contrib-all>`.
 
-Start on Login
-~~~~~~~~~~~~~~
+Run at user log on using the Startup folder
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Starting Syncthing at user log on, without a console window or browser
+opening on start, is relatively easy.
+
+#. Copy and paste ``%APPDATA%\Microsoft\Windows\Start Menu\Programs\Startup``
+   into the Address Bar in Windows Explorer, and press Enter.
+
+   |Windows Startup Folder Address Bar Screenshot|
+
+   Alternatively, in newer versions of Windows, open ``shell:startup``
+   from the Run command (``Win+R``).
+
+#. Right-click empty space in the folder and choose "New", and then
+   "Shortcut".
+
+   |Windows Startup Folder New Shortcut Screenshot|
+
+#. Enter the path to ``syncthing.exe`` in "Type the location of the item:"
+   followed by ``-no-console -no-browser`` (for example ``C:\syncthing\syncthing.exe
+   -no-console -no-browser``)
```

Comment:
```suggestion
   -no-console -no-browser``).
```

Reply:
I clicked on the "Commit suggestion" button, and now we are both listed as the authors. Should I commit the change manually using git instead?